src/main/resources/avro/rnaquantification.avdl

```diff
@@ -0,0 +1,216 @@
+@namespace("org.ga4gh.models")
+/**
+This protocol defines feature expression counts on GA4GH reads. 
+*/
+protocol RnaQuantifications {
+
+import idl "common.avdl";
+import idl "reads.avdl";
+
+/**
+Units for expression level
+*/
+enum ExpressionUnits {
+  FPKM,
+  RPM
+}
+
+/**
+Top level identifying information
+*/
+record RnaQuantification {
+  /** 
+  The unique ID assigned to the results of running the described programs on the
+specified reads and assignment to the listed annotation.
+  */
+  string id;
+ 
+  /**
+  Name
+  */
+  union {null, string} name = null;
+
+  /**
+  Description
+  */
+  union {null, string} description = null;
+
+  /**
+  ID of the ReadGroup providing the reads for the analysis.
+  */
+  string readGroupId;
+
+  /**
+  List of programIds used in the analysis.
+  */
+  array<string> programIds = [];
+
+  /**
+  List of annotations used.
+  */
+  array<string> annotationIds = [];
+}
+
+/**
+Read characterization data.
+*/
+record Characterization {
+  /** 
+  The associated RnaQuantification. 
+  */
+  string analysisId;
+ 
+  /**
+  Complexity of the reads
+  */ 
+  float complexity;
```

Comment:
Sean Upchurch <notifications@github.com> writes:
> 
> Complexity is just a percentage

percentage or fraction?  Please add to documentation